charts/rancher-cis-benchmark/2.0.4/Chart.yaml

```diff
@@ -2,12 +2,12 @@ annotations:
   catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: CIS Benchmark
-  catalog.cattle.io/kube-version: '>=1.16.0-0'
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
release in 2.6.5 based on https://github.com/rancher/rancher/issues/37376, this does not look correct.

Tested versions are 1.18-1.23, not ending at 1.21, based on the support matrix in https://www.suse.com/assets/EN-2.6.5SupportMatrix-300422-0116-26.pdf, but 1.16-1.17 are fine to keep in the range.

**Needs changes**: kube-version for all charts released in 2.6.5 are too narrow in range since they should include 1.22 and 1.23 support